tests/Unit/PropertyBuilder/RelatedProductsPropertyBuilderTest.php

```diff
@@ -13,16 +13,15 @@
 use BitBag\SyliusCrossSellingPlugin\PropertyBuilder\RelatedProductsPropertyBuilder;
 use Doctrine\Common\Collections\ArrayCollection;
 use Elastica\Document;
-use FOS\ElasticaBundle\Event\TransformEvent;
+use FOS\ElasticaBundle\Event\PostTransformEvent;
 use PHPUnit\Framework\TestCase;
 use Sylius\Component\Core\Model\OrderInterface;
 use Sylius\Component\Core\Model\OrderItemInterface;
 use Sylius\Component\Core\Model\ProductInterface;
 
 final class RelatedProductsPropertyBuilderTest extends TestCase
 {
-    /** @var RelatedProductsPropertyBuilder */
-    private $sut;
+    private RelatedProductsPropertyBuilder $sut;
```

Comment:
Maybe better would be to rename it, as the `$sut` doesn't tell me anything about it? :)